File: Gdx/Historic/fell/core/src/com/dugsolutions/fell/map/gen2/IGenZ.java
package com.dugsolutions.fell.map.gen;

import com.badlogic.gdx.math.Vector3;

public interface IGenZ {
	void addHeight(int x, int y, float z);

	void addNormal(int x, int y, Vector3 n);

	boolean genNormal();
}

File: Gdx/FellDungeon/app/src/main/java/com/dugsolutions/fell/map/gen2/IGenZ.kt
package com.dugsolutions.fell.map.gen2

import com.badlogic.gdx.math.Vector3

interface IGenZ {
    fun addHeight(x: Int, y: Int, z: Float)
    fun addNormal(x: Int, y: Int, n: Vector3)
    fun genNormal(): Boolean
}